### PR TITLE
Tidy up help docs

### PR DESCRIPTION
**Pull request by GodloveD**
Making sure that docs are accurate, updated from the 2.x series, and that formatting is consistent across all commands and flags

### CODE REVIEW

File: src/cmd/singularity/cli/build.go [hunk 1]
```diff
@@ -39,37 +39,34 @@ var buildflags = pflag.NewFlagSet("BuildFlags", pflag.ExitOnError)
...
-	BuildCmd.Flags().BoolVarP(&force, "force", "F", false, "Delete and overwrite an image if it currently exists")
+	BuildCmd.Flags().BoolVarP(&force, "force", "F", false, "delete and overwrite an image if it currently exists")
 	BuildCmd.Flags().SetAnnotation("force", "envkey", []string{"FORCE"})
 
-	BuildCmd.Flags().BoolVarP(&update, "update", "u", false, "Run definition over existing container")
+	BuildCmd.Flags().BoolVarP(&update, "update", "u", false, "run definition over existing container (skips header)")
 	BuildCmd.Flags().SetAnnotation("update", "envkey", []string{"UPDATE"})
 
-	BuildCmd.Flags().BoolVarP(&noTest, "notest", "T", false, "Bootstrap without running tests in %test section")
+	BuildCmd.Flags().BoolVarP(&noTest, "notest", "T", false, "bootstrap without running tests in %test section")
```
Comment (bauerm97) [inline src/cmd/singularity/cli/build.go]: `"build without running tests in %test section"`

File: src/cmd/singularity/cli/push.go [hunk 2]
```diff
@@ -20,7 +20,7 @@ var (
 func init() {
 	PushCmd.Flags().SetInterspersed(false)
 
-	PushCmd.Flags().StringVar(&PushLibraryURI, "library", "https://library.sylabs.io", "")
+	PushCmd.Flags().StringVar(&PushLibraryURI, "library", "https://library.sylabs.io", "library to push to")
```
Comment (bauerm97) [inline src/cmd/singularity/cli/push.go]: For consistency with `singularity pull` this should probably be `"the library to push to"`

**Review by bauerm97 — CHANGES_REQUESTED** [review]
Couple minor changes for consistency

File: src/cmd/singularity/cli/build.go (hunk 1 above)
Comment (GodloveD) [reply src/cmd/singularity/cli/build.go]: nice catch

File: src/cmd/singularity/cli/push.go (hunk 2 above)
Comment (GodloveD) [reply src/cmd/singularity/cli/push.go]: Got it!

**Review by bauerm97 — APPROVED** [review]

**Pull request merged**